CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased 2.x]
 ### Added
-- [Extensions] Moving Extensions APIs to support cross versions via protobuf. ([#6960](https://github.com/opensearch-project/OpenSearch/pull/6960))
+- [Extensions] Moving Extensions APIs to support cross versions via protobuf. ([#7402](https://github.com/opensearch-project/OpenSearch/issues/7402))
```

Comment:
Can we please wait on it until #7408 gets merged in? 
Or you can drop the CHANGELOG and that will take care of it.

Reply:
Sure. Either way works.

Reply:
I think you are better off dropping the file. Saves on the merge conflict :) 
#7408 got merged in with this change.